jupyter-book/conditions/perturbation_modeling.ipynb

```diff
@@ -22,12 +22,12 @@
    "metadata": {},
```

Comment:
<p>I don't think this makes sense - it's because Augur performs inference on the cell type level that you should be able to use these genes as markers of the perturbation, and hence find it slightly different from differentially expressed genes.</p><p> </p><p>Meanwhile, I also think you should just delete this paragraph because it's not your responsibility to justify or not this usage.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/91/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>